ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6605,4 +6605,52 @@ VMADC:
   - mixed_air_damper_percentage_command
   - mixed_air_damper_percentage_sensor
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+SFPC:
+  description: "supply pressure monitoring, flow control."
+  is_abstract: true
+  opt_uses:
+  - supply_fan_run_command
+  - supply_air_linearvelocity_sensor
+  - supply_fan_rotationalvelocity_sensor
+  uses:
+  - supply_air_static_pressure_sensor
+  - supply_air_flowrate_sensor
+  - supply_air_flowrate_setpoint
+  implements:
+  - OPERATIONAL
+
+EFPC:
```

Comment:
Can we instead use `ED`?

Reply:
We can use ED if i set the damper command as missing and then add the `supply_fan_run_command`
  `supply_air_linearvelocity_sensor
  `supply_fan_rotationalvelocity_sensor` somewhere else?

Reply:
@kevin-hereworks yes please use `ED`

1. Use `supply_fan_run_command` by replacing `SS` with `SFSS`
2. Use existing field `supply_air_flowrate_sensor` and delete `supply_air_linearvelocity_sensor`
3. Add `supply_fan_rotationalvelocity_sensor` as optional on `SFVSC` (and make certain fields on that type missing, if needed)

Reply:
okay and then can the fire and smoke alarms be added to the sfss?

Reply:
I would suggest excluding the fire and smoke alarms from this type. Typically, we see those fields modeled as part of a SAFETY system.